flink-libraries/flink-table/src/main/java/org/apache/calcite/avatica/util/DateTimeUtils.java

```diff
@@ -802,6 +802,8 @@ public static int unixTimeExtract(TimeUnitRange range, int time) {
 			case SECOND:
 				final int seconds = time / (int) MILLIS_PER_SECOND;
 				return seconds % 60;
+			case MICROSECOND:
```

Comment:
we do not have to change this class as it will be removed from Flink after bumping calcite version as the result of the ticket https://issues.apache.org/jira/browse/FLINK-7235

Please use avatica 12 and https://github.com/apache/calcite/pull/702 instead
In this way you will not have to do such changes (just remove this class from your version to be sure everything works fine)

Reply:
Should I add dependency from from your branch? What tool you can advice?